contents/navigation/navbars/navbar-with-search/index.tsx

```diff
@@ -0,0 +1,190 @@
+import type { FC } from "react"
+import {
+  ChevronRightIcon,
+  LightbulbIcon,
+  PlusIcon,
+  SearchIcon,
+  SquareCheckBigIcon,
+  UserIcon,
+} from "@yamada-ui/lucide"
+import {
+  Avatar,
+  Box,
+  Divider,
+  HStack,
+  IconButton,
+  Indicator,
+  Input,
+  InputGroup,
+  InputLeftElement,
+  InputRightElement,
+  Kbd,
+  LinkBox,
+  LinkOverlay,
+  Spacer,
+  Text,
+  VStack,
+} from "@yamada-ui/react"
+
+const NavbarWithSearch: FC = () => {
+  const tabs = {
+    collections: [
+      { emoji: "👍", label: "Sales" },
+      { emoji: "🚚", label: "Deliveries" },
+      { emoji: "💸", label: "Discounts" },
+      { emoji: "💰", label: "Profits" },
+      { emoji: "✨", label: "Reports" },
+      { emoji: "🛒", label: "Orders" },
+      { emoji: "📅", label: "Events" },
+      { emoji: "🙈", label: "Debts" },
+      { emoji: "💁‍♀️", label: "Customers" },
+    ],
+    general: [
+      { icon: LightbulbIcon, label: "Activity", notifications: "3" },
+      { icon: SquareCheckBigIcon, label: "Tasks", notifications: "4" },
+      { icon: UserIcon, label: "Contacts", notifications: "" },
+    ],
+  }
+
+  return (
+    <VStack
+      as="nav"
+      borderRight="1px solid"
+      borderRightColor="border"
+      h="full"
+      minH="700px"
+      w="300px"
+    >
+      <LinkBox
+        as="article"
+        cursor="pointer"
+        maxW="sm"
+        transition="all 0.1s"
+        _hover={{
+          background: ["#F5F5F5", "gray.800"],
+          textDecor: "none",
+        }}
+      >
+        <LinkOverlay href="#" onClick={(event) => event.preventDefault()}>
+          <HStack gap="md" p="md">
+            <Avatar
+              name="Hirotomo Yamada"
+              src="https://avatars.githubusercontent.com/u/84060430?v=4"
+            />
+            <HStack gap="md">
+              <Box>
+                <Text as="b" fontSize="sm">
+                  Hirotomo Yamada
+                </Text>
+                <Text color="gray.400" fontSize="xs">
+                  hirotomo.yamada@avap.co.jp
+                </Text>
+              </Box>
+              <ChevronRightIcon />
+            </HStack>
+          </HStack>
+        </LinkOverlay>
+        <Divider variant="solid" />
+      </LinkBox>
+
+      <VStack as="nav" px="md">
+        <InputGroup size="sm">
+          <InputLeftElement>
+            <SearchIcon color="gray.500" />
+          </InputLeftElement>
+          <Input
+            color={["#505050", "gray.200"]}
+            fontSize="xs"
+            placeholder="Search"
+            pr="4.6rem"
+          />
+          <InputRightElement h="1.85rem" w="4.6rem">
+            <Text color={["black", "white"]}>
+              <Kbd>Ctrl + K</Kbd>
+            </Text>
```

Comment:
I set the height for the element to make it align (screenshot below), not sure if this is okay or not, I tried `align: center` but it doesn't seem work
<img width="311" alt="Screenshot 2024-10-22 at 11 58 56 PM" src="https://github.com/user-attachments/assets/0c7ff99e-dc71-49ab-814b-9b7f4d8db1d0">

Reply:
Hmm, since position: relative; is valid for InputGroup, how about adjusting the position with top/left of InputRightElement?
If that doesn't work, let's use the fixed height method.

Reply:
Thank you for your suggestion! I tried using `position: relative`, but it still left a small gap above the Kbd element. To resolve this, I kept the fixed height, but I still followed your advice to ensure that the `InputRightElement` stays within the bounds of the `InputGroup`!